common-codegen/src/main/java/io/vrap/rmf/codegen/common/generator/extensions/types/StringTypeExtension.java

```diff
@@ -0,0 +1,29 @@
+package io.vrap.rmf.codegen.common.generator.extensions.types;
+
+import io.vrap.rmf.codegen.common.processor.annotations.ExtensionMethod;
+import io.vrap.rmf.codegen.common.processor.annotations.ModelExtension;
+import io.vrap.rmf.raml.model.types.StringInstance;
+import io.vrap.rmf.raml.model.types.StringType;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@ModelExtension(extend = StringType.class)
+public class StringTypeExtension {
+
+
+    @ExtensionMethod
+    boolean isEnumeration(StringType stringType){
```

Comment:
We should add this method to RMF and I created this issue: https://github.com/vrapio/rest-modeling-framework/issues/26

Reply:
Nice its actually useful to add to the rmf framework.